views/post.tt

```diff
@@ -47,12 +45,14 @@
                     </div>
                     
                     <!-- Comment posting -->
-                    [% IF ! settings.static %]
-                    [% INCLUDE comment_form.tt %]
+                    [% IF ! settings.static and comments_engine.comment_form_tt -%]
```

Comment:
Does this `! settings.static` make sense here if the comments_engine is a third party service? (Disqus, IntenseDebate, etc)